core/activity_pub.md

```diff
@@ -0,0 +1,52 @@
+
+The latter requires the concept of communal ownership, which Activity Pub lacks. Suppose an Activity Pub implementation added support for DID's and thus fulfilled the requirements for the right to individual exit. In the presence of a malicious operator, we can reasonably predict that even in the case of an organized exit, the size and network effect of the forked community will be less than the original community. The right to collective exit and implicitly the right to partial ownership of a community helps mitigates the risk of malicious operators while aligning the incentives of operators, producers, moderators, and consumers.
+
+
+## URI and DID
+
+URIs (Uniform Resource Identifier) are used within Activity Pub to identify different servers and clients.  
+
+Fora will use [decentralized identifiers](https://www.w3.org/TR/did-core/), to identify both members and communities. 
+
+Please refer to the DID specification for further details, but it is essential to understand that:
+- a DID is a URI (it can be a URL and it has properties such as Query, Fragment etc)
+- a DID identifies a DID Subject, and that subject must never change
+- a DID resolves to a DID Document which describes the DID subject
+- a DID Document may be modified by one or more DID Controllers
+- a DID Subject is not necessarily also the DID Controller
+- there exists a verification method for each DID controller, which is specified in the DID Document
+- there exists an authentication method to be associated with a DID Subject, which is specified in the DID Document
+- All information may be modified in a DID Document except the id of the subject
+
+The DIDs of a client must be recorded on a verifiable data registry, typically this will be a public blockchain or ledger. All communities will rely on client DIDs to resolve the authentication method used by a particular client (public key the client signs ActivityPub messages with). Communities may choose not to rely on DIDs for authentication of governance module interaction. If the communities choose to use on-chain DID verification, the client DID controllers must be verifiable via a communication protocol such as InterBlockchain Communication. It is recommended that governance modules support using a third verification relationship known as `voting authentication` to authenticate any votes casted by a DID controller on behalf of a DID subject. If the `voting authentication` verification relationship does not exist, governance modules should default to using the authroization method of DID controllers to verify casted votes. If communities do not want to require on-chain verifiable DIDs for each member, members must register with the governance module a public key that they will use to cast votes.
```

Comment:
```suggestion
The DIDs of a client must be recorded on a verifiable data registry, typically this will be a public blockchain or ledger. All communities will rely on client DIDs to resolve the authentication method used by a particular client (public key the client signs ActivityPub messages with). Communities may choose not to rely on DIDs for authentication of governance module interaction. If the communities choose to use on-chain DID verification, the client DID controllers must be verifiable via a communication protocol such as InterBlockchain Communication. It is recommended that governance modules support using a third verification relationship known as `voting authentication` to authenticate any votes cast by a DID controller on behalf of a DID subject. If the `voting authentication` verification relationship does not exist, governance modules should default to using the authorization method of DID controllers to verify casted votes. If communities do not want to require on-chain verifiable DIDs for each member, members must register with the governance module a public key that they will use to cast votes.
```